charts/rancher-csp-billing-adapter/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: rancher-csp-billing-adapter
+description: Cloud Service Provider Marketplace Billing Adapter for Rancher.
+version: 999
+appVersion: 999
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: cattle-csp-billing-adapter-system
+  catalog.cattle.io/release-name: csp-rancher-billing-adapter
+  catalog.cattle.io/display-name: "CSP Rancher Billing Adapter"
+  catalog.cattle.io/scope: management
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux
+  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
+  catalog.cattle.io/kube-version: '>= 1.22.0-0 < 1.27.0-0'
```

Comment:
You might want to double-check some of these annotations. If this is a hidden chart/package than you may not need filtering for the versions this way.

Reply:
Can you please elaborate on that? I am not familiar with the concept of hidden chart/package.